src/main.rs

```diff
@@ -1,13 +1,44 @@
+use std::collections::HashMap;
+use std::env;
 use rasta_rs::RastaListener;
-use sci_rs::scils::{SCILSBrightness, SCILSSignalAspect};
+use sci_rs::scils::{SCILSBrightness, SCILSMain, SCILSSignalAspect};
 use sci_rs::SCIListener;
 use sci_rs::SCIMessageType;
 use sci_rs::SCITelegram;
 use std::net::SocketAddr;
-
+use std::path::Path;
+use config_file::FromConfigFile;
 mod oc_interface;
+use serde::{Serialize, Deserialize};
+
+#[derive(Serialize, Deserialize, Debug, Clone)]
+pub struct PinConfig {
+    number_of_pins: usize,
+    pins:   Vec<String>,
+    signals: HashMap<String, Vec<u8>>,
+}
+
+impl ::std::default::Default for PinConfig {
```

Comment:
Maybe `use std::default::Default` so you can just `impl Default for`.

Actually, do we even need to import `Default` or does that happen automatically?